File: src/main/java/us/n8l/duplicatefinder/digests/MD5.java
package us.n8l.duplicatefinder.digests;

import org.jetbrains.annotations.NotNull;

public class MD5 extends BaseDigest {
    public MD5(@NotNull byte[] bytes) {
        super(bytes);
    }

    @Override
    protected Algorithm getAlgorithm() {
        return Algorithm.MD5;
    }
}

File: src/main/java/us/n8l/duplicatefinder/digests/MD5.kt
package us.n8l.duplicatefinder.digests

import java.io.InputStream

class MD5 : BaseDigest {
    constructor(bytes: ByteArray) : super(bytes)

    constructor(inStream: InputStream) : super(inStream)

    override fun getAlgorithm() = Algorithm.MD5
}
